lib/Conversion/TorchToLinalg/Linear.cpp

```diff
@@ -127,8 +127,15 @@ class ConvertAtenMmOp : public OpConversionPattern<AtenMmOp> {
               "mismatching contracting dimension for torch.aten.mm"));
     }
 
+    auto resultTy = op.getType().cast<ValueTensorType>();
+    auto resultDTy = resultTy.toBuiltinTensor().getElementType();
     Type newResultType = getTypeConverter()->convertType(op.getType());
     Type elementType = newResultType.cast<TensorType>().getElementType();
+    auto accumulatorDType = getDefaultAccType(rewriter, resultDTy);
+    if (!isa<mlir::NoneType>(accumulatorDType) &&
```

Comment:
See comment in other PR - if accumulatorDType is none you should set it to the result type rather than repeating this check everwhere.